app/assets/stylesheets/administrate/components/_field-unit.scss

```diff
@@ -0,0 +1,17 @@
+.field-unit {
+  @include clearfix;
+  @include fill-parent;
+  align-items: center;
+  display: flex;
+  margin-bottom: $base-spacing;
+  position: relative;
+}
+
+.field-unit__label {
+  @include span-columns(2 of 12);
```

Comment:
Great point! I’m actually leaning towards something like; in fact, I’ve been looking at maybe just removing Neat all together, because this is really the only place it’s used.

For now, I just wanted to get this refactored and come out with minor visual adjustments at the end. Another PR could be visual improvements, which lead to some smarter flexbox usage.